src/utils.js

```diff
@@ -1,5 +1,23 @@
-const fetchWeather = cityName => {
-  const API_KEY = "<INSERT_API_KEY>";
+const fetchWeather = async (cityName) => {
+  const API_KEY = process.env.REACT_APP_WEATHER_API_KEY;
   // TODO: fetch weather forecast from endpoint
   // from https://openweathermap.org/api
+  const URL = `http://api.openweathermap.org/data/2.5/weather?q=${cityName}&APPID=${API_KEY}&units=imperial`
+  let dataObj = {}
+  await fetch(URL)
+  .then(res =>  res.json())
+  .then(data => {
```

Comment:
Yes there is! One way to do it is by creating a constant that holds in that value for that asynchronous function with the use of async await. So instead of the chaining i can just do 
const res = await fetch(URL)
const data = await res.json()
and then be able to use the fetched data synchronously and be able to let javascript do its thing. 
 Thank you for catching that for me!